### ISSUE TITLE
Proxmox distro detection

### ISSUE THREAD

**Issue by M0NsTeRRR** [KittyKatt/screenFetch#656]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] new distro request
```

**Bug report**
```
=> search Github for a similar issue or PR before submitting
=> download the latest revision from Github and check if the bug is still present
   -> https://github.com/KittyKatt/screenFetch/raw/master/screenfetch-dev
=> Show us the output of: screenfetch -v
```
![screen](https://user-images.githubusercontent.com/37785089/70392330-48eb4100-19df-11ea-9911-c8a84f91b8bb.PNG)

**Comment by M0NsTeRRR** [comment]
Fix #655

**Comment by M0NsTeRRR** [comment]
thanks you @darealshinji
